tests/system/common.py

```diff
@@ -763,3 +763,11 @@ def task_by_name(tasks, name):
             return task
 
     assert False, "Did not find task with name %s in this list of tasks: %s" % (name, tasks,)
+
+
+@retrying.retry(wait_fixed=100, stop_max_attempt_number=30, retry_on_exception=ignore_exception, stop_max_delay=30000)
```

Comment:
is 3s enough?

Reply:
It's 30 seconds.

Reply:
if we do up to 30 retries every 100 ms it might be just 3s, right?

Reply:
Ah. You a right. I looked at the max wait time.